cves/CVE-2018-13304.yml

```diff
@@ -263,10 +271,12 @@ interesting_commits:
       * A change that looks really sketchy
       * Commits where a developer mentions security in their message or comments
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 4d2858deac5213eaddfdc06f98379b6325d7b953
+    note: |
+      Added the error resistance file which started the standard for 
+      being more careful with profiles, and moved a lot of checks to this new file.
```

Comment:
Cool finds! Could you elaborate on what `error resistance is`? I think this would make this CVE and your writeup stand out even more.